python/test/test_cylon_front.py

```diff
@@ -0,0 +1,18 @@
+from pycylon.data.table import csv_reader
+from pycylon.data.table import Table
+from pycylon.ctx.context import CylonContext
+
+ctx: CylonContext = CylonContext("mpi")
+
+tb1: Table = csv_reader.read(ctx, '/tmp/csv1.csv', ',')
+
+tb2: Table = csv_reader.read(ctx, '/tmp/csv2.csv', ',')
+
```

Comment:
shall we not have /tmp/csv1.csv, lets generate it or take it as a command line arg